src/runtime_src/core/tools/xbmgmt2/ReportPlatform.cpp

```diff
@@ -230,9 +233,13 @@ ReportPlatform::writeReport( const xrt_core::device * device,
   output << std::endl;
 
   output << "Flashable partitions running on FPGA\n";
-  output << fmtBasic % "Platform" % pt.get<std::string>("platform.current_shell.vbnv", "N/A");
-  output << fmtBasic % "SC Version" % pt.get<std::string>("platform.current_shell.sc_version", "N/A");
-
+  if (is_mfg) {
+  	output << fmtBasic % "Platform" % board;

```

Comment:
no tabs should be used for spacing, please fix that, indentation should be of 2 spaces